src/formApps/_AutorecMenu/store/default-data/aefx.js

```diff
@@ -8,7 +8,7 @@ export const aefx = [
       activeEffectType: "ontoken",
       primary: {
          options: {
-            addTokenWidth: false,
+            addTokenWidth: true,
```

Comment:
This change makes it so that larger tokens (or smaller too I suppose) don't have a "tiny" animation floating in the middle of their token but instead use the full token size.